src/main/java/frc/robot/Constants.java

```diff
@@ -31,10 +31,31 @@ public static class PathConstants {
     public static final PathConstraints CONSTRAINTS = new PathConstraints(
             0.3, 1, 
             Units.degreesToRadians(540), Units.degreesToRadians(720));
+
+    public static final Pose2d NOTE_1_POSE = new Pose2d(2.9, 4.1, Rotation2d.fromDegrees(90));
+    public static final Pose2d NOTE_2_POSE = new Pose2d(2.9, 5.55, Rotation2d.fromDegrees(90));
+    public static final Pose2d NOTE_3_POSE = new Pose2d(2.9, 7, Rotation2d.fromDegrees(90));
+    public static final Pose2d CENTER_NOTE_1_POSE = new Pose2d(8.29, 0.76, Rotation2d.fromDegrees(90));
+    public static final Pose2d CENTER_NOTE_2_POSE = new Pose2d(8.29, 2.45, Rotation2d.fromDegrees(90));
+    public static final Pose2d CENTER_NOTE_3_POSE = new Pose2d(8.29, 4.1, Rotation2d.fromDegrees(90));
+    public static final Pose2d CENTER_NOTE_4_POSE = new Pose2d(8.29, 5.77, Rotation2d.fromDegrees(90));
+    public static final Pose2d CENTER_NOTE_5_POSE = new Pose2d(8.29, 7.44, Rotation2d.fromDegrees(90));
+
     public static final Pose2d STATION_1 = new Pose2d(0.4119143784046173, 7.161474227905273, Rotation2d.fromRotations(0));
     public static final Pose2d STATION_2 = new Pose2d(0.5068893432617188, 3.710716009140014, Rotation2d.fromRotations(0));
     public static final Pose2d STATION_3 = new Pose2d(0.44357267022132874, 2.3525, Rotation2d.fromRotations(0));
   }
+
+  public static class AutoConstants {
+    public static final Pose2d FIRST_NOTE_SHOOTING_POSITION = new Pose2d(0, 0, Rotation2d.fromRotations(0));
```

Comment:
Try to find an actual shooting position for this. I suspect it might just be a little bit in front of the first note pickup pose.